ce/api/data.py

```diff
@@ -112,6 +113,8 @@ def get_spatially_averaged_data(data_file, time_idx):
         :param time_idx (int): index of time of interest
         :return: float
         """
+        if "/storage" in data_file.filename:
+            data_file.filename = os.getenv("THREDDS_URL_ROOT") + data_file.filename
```

Comment:
You added a a new `is_thredds` parameter to the `stats` API, but not to this `data` API. It looks like it always tries to use thredds. If so, I don't think that is behaviour we want, we probably want the server supporting the PCEX web portal to be able to continue to access files locally, even though we also want a version to be able to work with thredds data.